components/sidebar/sidebar_next_steps/close_next_steps_modal.scss

```diff
@@ -0,0 +1,68 @@
+.CloseNextStepsModal__helpBox {
+    position: fixed;
+    top: 16px;
+    left: 201px;
+    z-index: 1050;
+}
+
+.CloseNextStepsModal__helpText {
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    text-align: center;
+    color: var(--center-channel-bg);
+    display: block;
+    width: 188px;
+    margin-left: -12px;
+    margin-top: 4px;
+}
+
+.modal .CloseNextStepsModal {
+    max-width: 514px;
+
+    .GenericModal__header {
+        text-align: center;
+        padding-bottom: 8px;
+
+        h1 {
```

Comment:
3/5 prefer the use of class names to avoid clashes.

Reply:
It's wrapped in a class for this modal already, so unless we have 2 `h1`s in the header, this can't clash with anything.